README.md

```diff
@@ -0,0 +1,243 @@
+direction Sawtooth is evolving in.
+
+This process is intended to be substantially similar to the Rust RFCs process,
+customized as necessary for use with Sawtooth. The README.md and
+000-template.md were initially forked from [Rust
+RFCs](https://github.com/rust-lang/rfcs).
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#sawtooth-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Sawtooth or any of its sub-components including but not limited to Sawtooth
+Core, Sawtooth Supply Chain, Sawtooth Seth, the various Sawtooth SDKs, or the
+RFC process itself. What constitutes a "substantial" change is evolving based
+on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Architectural changes
+  - Substantial changes to component interfaces
+  - New core features
+  - Backward incompatible changes
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism, trap
+    more errors, etc.)
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+
+### Sub-team specific guidelines
+[Sub-team specific guidelines]: #sub-team-specific-guidelines
+
+For more details on when an RFC is required for the following areas, please see
+the Sawtooth community's [sub-team] specific guidelines for:
+
+
+  - [core changes](core_changes.md)
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on [#sawtooth](https://chat.hyperledger.org/channel/sawtooth) and
```

Comment:
Links are inconsistent.  Most, like these, are "hidden", but the one for the Hyperledger Sawtooth mailing list appears in formatted contents.  In general, I prefer showing the links explicitly (along with making them active links).  But mostly, I just like consistency.

Reply:
I don't want to churn on this too much, because I think we should actually link to a community page for all of these things instead of having them directly in this repository.  I don't want to link to the core docs for this though, but rather a page we maintain in a sawtooth-website repository in the near future (which helps manage sawtooth.hyperledger.org content).

Reply:
My comment was simply on how links appear when you view the readme in a browser.  Sometimes a link looks like this:

>  #sawtooth``

And one link looks like this: 

> Hyperledger Sawtooth mailing list (https://lists.hyperledger.org/mailman/listinfo/hyperledger-stl)``

My comment is only about the inconsistency of mostly "hiding" the URL but showing it in one link.